src/commands/ping.ts

```diff
@@ -0,0 +1,11 @@
+import { SlashCommand } from "../types/utils";
+import contents from "../contents/ping";
+
+export default {
+  name: "ping",
+  description: contents.DESCRIPTION,
+  production: false,
+  execute: async (_client, interaction) => {
+    await interaction.reply(contents.PONG);
+  },
+} as SlashCommand;
```

Comment:
Can we use this structure for multiple input? Like `CommandBuilder`

Reply:
Maybe `as SlashCommand` can change to `satisfies SlashCommand`

Reply:
Can, but I didnt copy everything to here
Example how it will looks like,
```ts
export default <SlashCommand>{
  name: SLASH_COMMAND.guild.reminder.name,
  description: SLASH_COMMAND.guild.reminder.description,
  commandName: SLASH_COMMAND.guild.name,
  type: 'subcommand',
  builder: (subcommand) =>
    subcommand
      .addRoleOption((option) =>
        option
          .setName('role')
          .setDescription('Select the role of the guild to update')
          .setRequired(true)
      )
      .addChannelOption((option) =>
        option.setName('channel').setDescription('Channel to send reminder message')
      )
      .addStringOption((option) =>
        option.setName('reminder-message').setDescription('Message to send when team raid is ready')
      ),
  execute: async (client, interaction) => {
```

Reply:
sure sure

Reply:
added btw